EIPS/eip-####.md

```diff
@@ -0,0 +1,124 @@
+
+The commit message backlinks to the issue, and the issue is closed as "completed" with a comment containing the corresponding receipt data. For each log, the GitHub action further adds a label to the issue for the log-emitting address and indexed topics.
+
+The GitHub issue template follows the usual transaction fields (including `To`, `Value`, `Input`), with the following differences.
+
+#### Signature
+
+No signature is needed. The `From` address is derived from the GitHub account opening the issue. The `Nonce` is obtained from the current state when the transaction executes. Transactions are executed in ascending issue number order.
+
+If a transaction fails as it lacks the required `BALANCE`, the issue is closed as "not completed".
+
+#### Fee market
+
+The base fee is set based on a moving average tracking the transaction execution time as given by the time difference of the GitHub issue being opened and it being closed by the system.
+
+There are no priority fees. Transactions are always executed in order of submission.
+
+Spam transactions are implicitly solved at the discretion of GitHub. Users creating lots of nonsensical GitHub issues MAY get banned and can contact GitHub support for followup.
+
+#### Access list
+
+Exhaustive access lists MUST be provided when submitting a transaction. The GitHub action executing the transaction will configure a Git sparse checkout based on the provided access list. Other state data will not be available to the GitHub actions runner.
+
+Transactions that access disjunct accounts and storage slots MAY always execute in parallel. Transactions that read the same data MAY execute in parallel only while no transaction writing to the data is processing.
+
+#### Blobs
+
+Blobs can be uploaded directly into the issue as a comment. A GitHub action periodically deletes them when they expire.
+
+#### Authorizations
+
+Authorization lists work as expected.
+
+However, to enable others to perform GitChain operations on behalf of the user, the authorizing user will have to submit an issue to create the authorization. Once that issue is processed, the user can prove that the authorization was created by computing a ZK proof over the confirmation email based on the GitHub DKIM signature. A new authorization type is required for other blockchains to consume GitChain authorizations.
+
+Alternatively, users MAY follow traditional OAuth flows to allow a third party to create an issue on their behalf.
+
+### Software upgrade
+
+The software can be upgraded by anyone opening a pull request on GitHub. The pull request MAY change any code and MAY include irregular state transitions. The PR gets automatically merged if more than 2/3rd of the total balance reacts with 👍 during a periodic check by GitHub actions.
```

Comment:
You should consider some form of quadratic voting where "🔥" counts as two votes.